starboard/build/config/BUILD.gn

```diff
@@ -238,3 +238,10 @@ config("default_compiler_flags") {
     asmflags = cflags
   }
 }
+
+# TODO: b/329683364 - Add non third party cpp standard flags.
```

Comment:
Could you rephrase this comment? I.e. "Move other flags setting the c++ standard from each platform's platform_configuration to this config."